freebridge-ai/recommendation.py

```
@@ -0,0 +1,54 @@
+router = APIRouter(prefix="/api/v1/employer", tags=["Recommendation"])
+
+@router.get("/jobs/{jobId}/recommendations", response_model=EmployerRecommendationResponse)
+async def get_job_recommendations(jobId: int):
+    try:
+        # 1. 벡터 스토어 및 LLM 설정
+        vectorstore = get_vectorstore()
+        llm = ChatUpstage(api_key=os.getenv("UPSTAGE_API_KEY"))
+        
+        # [핵심] Structured Output 설정 (미사여구 차단)
+        # 추천 사유는 빼고 FreelancerMatch 리스트만 받도록 강제함
+        structured_llm = llm.with_structured_output(List[FreelancerMatch])
```

Ruff (0.15.2)

[error] Undefined name `List`

(F821)